commisery/commit.py

```diff
@@ -175,7 +225,7 @@ class ConventionalCommit(CommitMessage):
 
     # 9.  A footer's token MUST use `-` in place of whitespace characters, e.g. `Acked-by` (this helps differentiate
     #     the footer section from a multi-paragraph body). ...
-        \w+(?:-\w+)*
+    [\w][\w\- ]+
```

Comment:
For the purpose of detecting it. See rule `C020`.
You'll notice this PR relaxes a lot of the strictness in the regexes, since the validation has been moved to `rules.py`.

Reply:
:+1: